### ISSUE TITLE
Types starting with __ are for internal use?

### ISSUE THREAD

**Issue by rushmorem** [gjtorikian/graphql-idl-parser#3]
First off congratulations on your release and thank you for this amazing contribution to the Rust community :clap: 

Maybe I'm missing something but [when I see](https://docs.rs/graphql-idl-parser/0.1.1/graphql_idl_parser/gqlidl/index.html) types like `__ToTriple` and functions like `__action0` I think these are implementation details not meant for use by third parties. Am I correct in assuming this or are they actually part of the API?

**Comment by gjtorikian** [comment]
Correct! Seems like these are artifacts from the [lalrpop] state machine(https://github.com/nikomatsakis/lalrpop) ? At any rate, I'll clean that up!
